plugins/copr.py

```diff
@@ -217,7 +217,7 @@ def run(self):
         try:
             chroot = self.opts.arg[1]
         except IndexError:
-            chroot = self._guess_chroot(self.chroot_config)
+            chroot = self._guess_chroot(self)
```

Comment:
the self as an argument is not needed, and it is there three times below as well.